### PR TITLE
docs(platform): datetime picker form submission returns form value

### PR DESCRIPTION
**Pull request by ASokolov22**
#### Please provide a link to the associated issue.
Fixes https://github.com/SAP/fundamental-ngx/issues/4466

#### Please provide a brief summary of this pull request.
Fixes problem with datetime picker form submission alert, returns form value instead of [object, object]

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by N1XUS — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export class PlatformDatetimePickerReactiveExampleComponent {
 
     onSubmit(): void {
         if (this.datetimePickerForm.valid) {
-            alert('Form Value: ' + this.datetimePickerForm.value);
+            alert('Form Value: ' + JSON.stringify(this.datetimePickerForm.value));
```
Comment (N1XUS) [inline apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts:31]: If we take into concideration other components, they usualy print form or control value into some paragraph. Maybe this should be done in the same way for the sake of consistency?

**Review by N1XUS — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts (hunk 1 above)
Comment (ASokolov22) [reply apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts:31]: Ok. Maybe I can find somewhere an example?

Comment (N1XUS) [reply apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts:31]: https://fundamental-ngx.netlify.app/#/core/checkbox#checkbox-reactive-forms-example

Comment (ASokolov22) [reply apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts:31]: Thanks! I'll fix it

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
@ASokolov22 changed the PR title to `docs` as it only touches the documentation.

**Comment by nikvalor** [comment]
Bug still actual
photo5330179415520425169

**Comment by ASokolov22** [comment]
> Bug still actual

Are we talking about the same component? I got rid of alert box and it looks like that:
Screenshot 2021-06-09 at 13 07 48

**Pull request merged**